scripts/wasm2js.js

```diff
@@ -191,6 +191,80 @@ var asmLibraryArg = {
     console.log('store_val_f64 ' + [loc, value]);
     return value;
   },
+
+  struct_get_val_i32: function(loc, value) {
+    console.log('struct_get_val_i32 ' + [loc, value]);
+    return value;
+  },
+  struct_get_val_i64: function(loc, value) {
+    console.log('struct_get_val_i64 ' + [loc, value]);
+    return value;
+  },
+  struct_get_val_f32: function(loc, value) {
+    console.log('loastruct_get_val_i32d_ptr ' + [loc, value]);
```

Comment:
All these `i32d_ptr` should be `f32`, right?

Reply:
Heh, good catch... looks like a very weird copy-paste error. Fixed.